proto/injective/txfees/v1beta1/txfees.proto

```
@@ -0,0 +1,64 @@
+syntax = "proto3";
+package injective.txfees.v1beta1;
+
+import "amino/amino.proto";
```

Buf (1.47.2)

import "amino/amino.proto": file does not exist

(COMPILE)